src/util/Util.js

```diff
@@ -242,11 +242,14 @@ class Util {
    * @private
    */
   static parseEmoji(text) {
-    if (text.includes('%')) text = decodeURIComponent(text);
-    if (!text.includes(':')) return { animated: false, name: text, id: null };
-    const m = text.match(/<?(?:(a):)?(\w{2,32}):(\d{17,19})?>?/);
-    if (!m) return null;
-    return { animated: Boolean(m[1]), name: m[2], id: m[3] || null };
+    if (text !== decodeURIComponent(text)) text = decodeURIComponent(text);
+    // eslint-disable-next-line max-len
```

Comment:
You should not disable max-len and instead split the statement onto 2 lines

Reply:
I shortened the regex down, it's 9 chars over now. Is disabling still not okay?